pkgs/development/libraries/qt-5/5.4/default.nix

```diff
@@ -152,7 +152,7 @@ let
 
       multimedia = callPackage
         (
-          { qtSubmodule, base, declarative
+          { qtSubmodule, base, declarative, pkgconfig
```

Comment:
That can matter for smone :)
Well, indeed that is used only in one pkg, so I think that making it optional is better.